tooling/Microsoft.VisualStudio.RazorExtension/DocumentInfo/RazorDocumentInfoWindowControl.xaml

```diff
@@ -29,6 +35,18 @@
             <Label Grid.Column="1" Grid.Row="2" Content="{Binding TextContainer}"/>
             <Label Grid.Column="0" Grid.Row="3" Content="Workspace"/>
             <Label Grid.Column="1" Grid.Row="3" Content="{Binding Workspace}"/>
+            <Label Grid.Column="0" Grid.Row="4" Content="Razor Lang"/>
+            <Label Grid.Column="1" Grid.Row="4" Content="{Binding RazorLanguageServices}"/>
+            <Label Grid.Column="0" Grid.Row="5" Content="Resolver"/>
+            <Label Grid.Column="1" Grid.Row="5" Content="{Binding TagHelperResolver}"/>
+            <Label Grid.Column="0" Grid.Row="6" Content="Syntax Facts"/>
+            <Label Grid.Column="1" Grid.Row="6" Content="{Binding RazorSyntaxFactsService}"/>
+            <Label Grid.Column="0" Grid.Row="7" Content="Template Eng"/>
+            <Label Grid.Column="1" Grid.Row="7" Content="{Binding RazorTemplateEngineFactoryService}"/>
+            <Label Grid.Column="0" Grid.Row="8" Content="Completion Service"/>
+            <Label Grid.Column="1" Grid.Row="8" Content="{Binding TagHelperCompletionService}"/>
+            <Label Grid.Column="0" Grid.Row="9" Content="TH Facts"/>
+            <Label Grid.Column="1" Grid.Row="9" Content="{Binding TagHelperFactsService}"/>
```

Comment:
To verify the correct services are flowing 😄. Mostly just used this to validate what I did worked. If we don't think it's valuable can nuke.